bindings/python/iota_sdk/types/block/metadata.py

```diff
@@ -7,25 +7,25 @@
 from typing import Optional
 from iota_sdk.types.common import HexStr, json
 from iota_sdk.types.block.block import Block
+from iota_sdk.types.transaction_metadata import TransactionMetadata
 
 
 @json
 @dataclass
 class BlockMetadata:
-    """Represents the metadata of a block.
+    """The metadata of a block.
     Response of GET /api/core/v3/blocks/{blockId}/metadata.
 
     Attributes:
-        block_state: The block state.
-        transaction_state: The transaction state.
-        block_failure_reason: The block failure reason.
-        transaction_failure_reason: The transaction failure reason.
+        block_id: TODO.
+        block_state: TODO.
+        block_failure_reason: TODO.
+        transaction_metadata: TODO.
```

Comment:
TODO

Reply:
I would need to come up with trivial docs though...In Rust we don't have them either, so I was wondering whether I should do that, or just remove the docs in Python as well. Wdyt?

Reply:
https://editor.swagger.io/?url=https://raw.githubusercontent.com/iotaledger/tips/tip48/tips/TIP-0048/openapi3-core.yaml has descriptions for the fields